cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/api/NodeController.java

```diff
@@ -479,6 +486,38 @@ public void write(final JsonGenerator json) throws IOException {
         });
     }
 
+    /**
+     * Update node with specified parameters. Note: node should be restarted for changes to take effect.
+     */
+    @POST
+    @Path("/{node}/rackdc")
+    public Response nodeRackDc(
+            @PathParam("node") String id,
+            @FormParam("rack") final String rack,
+            @FormParam("dc") String dc
+    ) {
```

Comment:
Can we change `rack` and `dc` to be a json object passed in the body rather than query params?

Something like this:
```json
{
  "dc": "DC_1",
  "rack": "rack_5"
}
```